R/helpers.R

```diff
@@ -449,19 +448,42 @@ create_dummy_package_full <- function(mn, title = "A Dummy Package") {
 #' \dontrun{
 #' pi_name <- get_orcid_name('https://orcid.org/0000-0002-2561-5840')
 #' }
+
 get_orcid_name <- function(orcid_url) {
-  req <- httr::GET(orcid_url)
+  req <- httr::GET(paste0(orcid_url, "/person.json"))
   if (req$status_code != 200) {
     stop('Failed to read in ', orcid_url)
   }
+  json <- httr::content(req)
+  return(json$displayName)
```

Comment:
This can probably fail on profiles without a display name.

Reply:
Yeah good call, I added a little fix and also exported the function